book.rb

```diff
@@ -0,0 +1,10 @@
+class Book
+  attr_reader :rental
+  attr_accessor :title, :author
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rental = []
+  end
```

Comment:
- [ ] As required in the project, Kindly make sure the students creat the `many-to-many` (also `has-many-through`) relationship between `Person` and `Book` using the intermediate class `Rental`
You may achieve this by saying:
```rb
def add_rental(person, date)
    # *the book itself should be sent as a parameter to create the new rental achieving the 'has-many' association. 
    Rental.new(date, self, person)
  end
```